04_numeronym/pfannerj/main.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var mainout io.Writer = os.Stdout
+
+func main() {
+
+	fmt.Fprintln(mainout, numeronyms("accessibility", "Kubernetes", "abc"))
+
+}
+
+func numeronyms(vals ...string) []string {
+
+	valsOut := vals
+	for i := range vals {
+		//Regular expression can be changed or removed altogether if additional chars need to be included
```

Comment:
I think this might be making the task a little more complicated that what was asked, as the lab never stated anything about ensuring the input was alphanumeric. So I think this isn't necessarily required, but if you want to proceed with this I am happy to let it go. Feel free to do either, just as long as the test cases reflect the implementation.

If you do keep it, I think the `regEx` variable should be moved so it is initialised outside the loop, rather than re initialising it each iteration, it will be constant for all.

Reply:
Cool.. I wasn't sure if it can really be called a Numeronym if it contains weird characters so I thought this was a good way to cater for that by just stripping them.  I think I will leave the regex in place but will defs move it outside the loop... thx!!